README.md

```diff
@@ -1,6 +1,44 @@
-# TO DO:
-## Deploy Hello-World App Engine
-## Create Cron Job, every 5 minutes,
-## Insert row to bigquery table (with current timestamp)
-## Upload file to cloud storage bucket (with current timestamp)
-## Update entity in Datastore (change timestamp to current timestamp)
+Workshop RBFA
+=============
+
+Introduction
+------------
+This repository was made during an internship at the Royal Belgian Football Association. The main goal of tis workshop is to learn how to use some basic functionalities of teh Google Cloud Platform like Bigquery, Storage and Datastore.
```

Comment:
it's not relevant to mention that this is an internship.

I would describe it more as an onboarding project